quickevent/app/quickevent/plugins/Competitors/src/competitorsplugin.cpp

```diff
@@ -72,8 +66,7 @@ int CompetitorsPlugin::editCompetitor(int id, int mode)
 void CompetitorsPlugin::onInstalled()
 {
 	qff::MainWindow *fwk = qff::MainWindow::frameWork();
-	m_partWidget = new ThisPartWidget();
-	fwk->addPartWidget(m_partWidget, manifest()->featureId());
+	qff::initPluginWidget<CompetitorsWidget, PartWidget>(tr("Competitors"), featureId());
```

Comment:
@fvacek please wait for resolving  #627 before releasing new version
@paukert in this branch are introduced new translated strings in the plugins like this, I think the words are already translated, but the translation files are not updated, please do some magic to fix this.

Reply:
Run the following command with correct paths, it will automatically update all TS files:
`QT_DIR/bin/lupdate-pro.exe -no-obsolete quickbox/quickbox.pro`

Reply:
the Translation files are still referenced in the .pro file though.
I have rewritten the strings to match the one in translations files and ran the update command.
The number of additions seems off (I do not know what is the issue here), but it works.

Reply:
I went quickly through CardReader.cs_CZ.ts and it contains also translations for Receipts plugin etc. Same problem occurs propably in every TS file :(

Reply:
So I guess we need to merge the files for each language into one?